build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
     id "java"
     id "maven-publish"
     id "org.ajoberstar.grgit" version "5.2.2"
-    id "com.matthewprenger.cursegradle" version "1.4.0"
+    id "com.matthewprenger.cursegradle" version "1.4.0" // This repository is archived on GH
```

Comment:
Why exactlly do we need this?

Reply:
This is to inform third-party contributors that this library is archived and that it isn't maintained anymore, hence why i left this note in there.